Src/ILGPU.Algorithms/Runtime/Cuda/API/NvmlAPI.xml

```diff
@@ -0,0 +1,609 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ImportLibrary Namespace="ILGPU.Runtime.Cuda.API"
+               ClassName="NvmlAPI"
+               VersionType="NvmlAPIVersion"
+               DefaultMethodAccessModifier="public"
+               DefaultReturnType="NvmlReturn">
+    <LibraryVersions>
+        <LibraryVersion Name="V6" Platform="Windows" LibName="nvml" />
+        <LibraryVersion Name="V6" Platform="Linux" LibName="nvml.so" />
+        <LibraryVersion Name="V6" Platform="OSX" LibName="nvml.dylib" />
```

Comment:
QUESTION: Is `nvml.so` and `nvml.dylib` the correct file names for Linux and OSX?

Reply:
The library bindings need to be adjusted:

- Windows: fine 👍 
- Linux: `nvidia-ml.so`
- OSX: `nvidia-ml.dylib`

Reply:
Thanks - bindings adjusted.